chesse20.java

```diff
@@ -0,0 +1,8 @@
+public class chesse20 {
+  public static void main (String [] args){
+  if username=chesse20
+    while true do
+    ++vbucks
+   else
+        end
+  }}
```

Comment:
this what it does

Reply:
Usless script.

Reply:
> .

thanks

Reply:
Can I get the script sent to me my epic games name is thebigkiwi101

Reply:
Get Fortnite Free V Bucks Hack Generator here===>>> https://freevbucksgenerators.pw
Get Fortnite Free V Bucks Hack Generator here===>>> https://freevbucksgenerators.pw

Reply:
You're actually retarded.

Reply:
> You're actually retarded.

Retarded is a social construct.